README.md

```diff
@@ -1,16 +1,76 @@
-# Language Recognizer
+# language-detect
+This is a language detection library using ML models
```

Comment:
Add: "Supports 800 languages out of the box and can be updated by users to support more languages."